### PR TITLE
Don't delete 3rd party license file

### PR DESCRIPTION
**Pull request by PhilETaylor**
Pull Request for Issue #22583 Part of #22219 .

### Summary of Changes

Do not delete 3rd party license file

### CODE REVIEW

File: administrator/components/com_admin/script.php [hunk 1]
```diff
@@ -1535,7 +1534,6 @@ public function deleteUnexistingFiles()
 			'/administrator/templates/isis/js/bootstrap.min.js',
 			'/administrator/templates/isis/js/jquery.js',
 			'/libraries/joomla/application/web/client.php',
-			'/libraries/simplepie/LICENSE.txt',
```
Comment (PhilETaylor) [inline administrator/components/com_admin/script.php]: This one is not really needed, as even in staging any longer as simplepie has been moved/removed

Comment (mbabker) [reply administrator/components/com_admin/script.php]: And that's why it's in the delete array 😉

(It was moved into Composer space so the files that used to be in `libraries/simplepie` had to be removed from existing sites, hence the lines here and below)

Comment (PhilETaylor) [reply administrator/components/com_admin/script.php]: yeah realised that after I added it - no harm done leaving this PR this way then.

**Comment by wilsonge** [comment]
Thanks Phil!

**Pull request merged**